### ISSUE TITLE
Display provided reference title in the description, not just e.g. DOI 1234/5678

### ISSUE THREAD

**Issue by StevenClontz** [pi-base/viewer#64]

**Comment by jamesdabbs-procore** [comment]
Agreed about this one, but it may be non-trivial to get working - we'd need 

* adapters for each link type that say how to fetch the title
* to keep those updated for any where we're having to scrape a page
* to make decisions about whether we're doing that at compile time and caching the resolved titles in the bundle vs. looking them up client-side

Definitely doable. But non-trivial.

**Comment by StevenClontz** [comment]
For clarity #65 is where I'm suggesting we use an API. Right now we store reference titles in the YAML metadata for many/most references.
